covalent_dispatcher/_cli/service.py

```diff
@@ -179,6 +195,46 @@ def _graceful_start(
     return port
 
 
+def _graceful_start_dask(
+    server_root: str,
+    pidfile: str,
+    logfile: str,
+    develop: bool = False,
+) -> Tuple[str, int]:
+    """
+    Gracefully start a Dask Cluster.
+
+    Args:
+        server_root: Directory where app_dask.py is located.
+        pidfile: Process ID file for the dask scheduler.
+        logfile: Log file for the dask scheduler.
+        port: Port requested to be used by the scheduler.
+        develop: Start the cluster in developer mode.
+
+    Returns:
+        addr: Address assigned to the scheduler.
+        port: Port assigned to the scheduler.
+    """
+
+    pid = _read_pid(pidfile)
+    if psutil.pid_exists(pid):
+        addr = get_config("dask.scheduler_address")
+        port = get_config("dask.scheduler_port")
+        click.echo(f"Dask scheduler is already running at {addr}.")
+        return addr, port
+
+    _rm_pid_file(pidfile)
+    pypath = f"PYTHONPATH={UI_SRVDIR}/../tests:$PYTHONPATH" if develop else ""
+    launch_str = f"{pypath} python app_dask.py >> {logfile} 2>&1"
+    proc = Popen(launch_str, shell=True, stdout=DEVNULL, stderr=DEVNULL, cwd=server_root)
```

Comment:
instead of routing the `stdout` and `stderr` to DEVNULL why not use PIPE, this way we can communicate with the process to check `stdout` and `stderr` logs